kover-gradle-plugin/src/functionalTest/kotlin/kotlinx/kover/gradle/plugin/test/functional/cases/GradleValidationTests.kt

```diff
@@ -0,0 +1,16 @@
+package kotlinx.kover.gradle.plugin.test.functional.cases
+
+import kotlinx.kover.gradle.plugin.test.functional.framework.checker.CheckerContext
+import kotlinx.kover.gradle.plugin.test.functional.framework.starter.TemplateTest
+import kotlin.test.assertFalse
+
+internal class GradleValidationTests {
+
+    // test on validation error https://github.com/gradle/gradle/issues/26018
```

Comment:
I assume this is a bug fixed in Gradle, why this test is needed?

Reply:
In case this error is reproduced in the future in Gradle, or incorrect `@Nested` classes are added in the plugin